### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by caroliine**
I'm not sure why it's making me do this. In the past, I've been able to save changes directly via github.

### CODE REVIEW

**Review by edanaher — APPROVED** [review]
Looks like we turned on the flag to require code reviews on this repo, so you need two folks to approve it.  I don't know if there's a way to turn off that flag for specific files and/or committers, or if we should just force you to go through the normal process.

And as long as I'm here, I'm curious if we do now have documents that use the nonWorkingNumberCode field and/or are no longer thinking of getting rid of it.

**Comment by edanaher** [comment]
@bradddd Hey, you want to give a second approval and merge this in?  I assume it's still accurate.

**Review by bradddd — APPROVED** [review]

**Comment by caroliine** [comment]
Thanks Steve.

Evan, FWIW, I deleted that section because we do in fact have documents containing both `nonWorkingNumberCode`, although they appear to be older than those with the `twilioCode` field

**Pull request merged**
